gaffer-core/common-util/src/main/java/gaffer/commonutil/StreamUtil.java

```diff
@@ -37,6 +37,9 @@
     public static final String STORE_TYPES = SCHEMA_FOLDER + "storeTypes.json";
     public static final String STORE_PROPERTIES = "/store.properties";
     public static final String OP_AUTHS = "/opAuths.properties";
+    public static final String OP_SCORES = "/opScores.properties";
+    public static final String ROLE_SCORES = "/roleScores.properties";
```

Comment:
It might be clearer to call these properties: "role max scores" or "max role scores". Just to ensure they are understood to be max scores and not scores that will be incremented.